README.md

```diff
@@ -4,7 +4,7 @@ Plugin for [Marathon](https://mesosphere.github.io/marathon/) which injects secr
 
 ## How to reference secrets in marathon.json
 
-The following example `marathon.json` fragment will read Vault path `/secret/abc/xyz`, extract field `password` from that path and inject the field value into an environment variable named `ENV_NAME`:
+The following example `marathon.json` fragment will read Vault path `/secret/shared/abc/xyz`, extract field `password` from that path and inject the field value into an environment variable named `ENV_NAME`:

```

Comment:
We should say somewhere that the `secret/shared` part comes from a configuration. Or maybe have separate examples for absolute and relative notations (and lose this example altogether).